### ISSUE TITLE
Russian speaking community meet-up in Minsk (Belorussia)

### ISSUE THREAD

**Issue by Igmat** [littledan/js-outreach-groups#6]
After some initial private discussions with @littledan we've decided to organize some kind of community group for collecting and providing feedback regarding existing proposal (ideally at early stages) to @tc39 committee.

My employer (@DataArt) agreed to sponsor first such meet-up for Russian-speaking community leaders in Minsk (Belorussia) in March 2019 (exact date isn't decided yet).

We expect to have 15-30 persons from different countries representing several companies. I assume that they will be mostly from out-source or B2B sector. This could lead us to a little bit different point of view, since it looks like that majority of committee members belongs to B2C companies.

## High-level agenda:
1. Small presentation explaining motivation of creating such community group (15-30 minutes, Russian, me)
    - also includes few words about how and why I get here (mentioning [class-fields-proposal](https://github.com/tc39/proposal-class-fields))
2. Talk about expectations of `tc39` committee using video conference (? minutes, English, @littledan)
3. Technical presentations (1 hour, me)
    1. Presentation of [realms](https://github.com/tc39/proposal-realms) _+ as an example of interaction between different proposals (class-fields in this case)_
    2. Presentation of [pipeline-operator](https://github.com/tc39/proposal-pipeline-operator) _+ as an example of good proposal process with competing alternatives_
    3. Presentation of [extended-numeric-literals](https://github.com/tc39/proposal-extended-numeric-literals) _+ as an example of one of the ways how community can help (I’ll probably finish babel plugin till that moment)_
    4. Questions and answers section
6. Presentation of possible framework for collaboration between community and committee (15-30 minutes, me)
7. Open discussion between all meeting participants

At the end, I believe, we'll reach some consensus on the best way possible for providing most valuable feedback.
I hope that few more artifacts could be created during this meeting (e.g. meetings schedule, list of volunteers that can present early proposals at conferences, etc.)

## Next steps:
 - [ ] create PR to this repo with agenda
 - [x] create google doc for collaborative editing of agenda
    https://docs.google.com/document/d/1wBbuOUuIGf_z6ok-g6AumM2oPrhhAVSs7txmOSrfQcQ/edit?usp=sharing
 - [ ] create PR to this repo with meeting notes (after actual meeting of course)
 - [ ] create PR with all valuable artifacts from meeting

## Questions
1. How committee will use such feedback?
2. How it'll affect decision making in `tc39`?
3. What should be done by community to bring the most valuable feedback?

I think that these 3 questions should be discussed before actual meeting, so we'll be able to discuss something more practical at event itself.
Also, I realize that second questions sound a little bit speculative, but we have to understand that nobody wants to waste time - so volunteers have to be sure that their efforts was at least taken into account. Knowledge of ways how it was analyzed will be also very valuable in terms of motivation.

## Additional notes
I believe that @hax is capable of doing something similar in Chinese community. @hax, how do you feel about that?
I assume that such events could be effective in terms of #4.
And it seems that we'll discuss what we can do to gather and analyze feedback (related to #5) at this meeting, but it would be great to have something more practical to start from.

**Comment by littledan** [comment]
I am really excited about this all. Can I suggest trying to cover 1-2 technical topics in the first meeting, to avoid packing it too much? Whichever ones you are most interested in would be good. (I have the same impulse to try to cover everything!)

**Comment by Igmat** [comment]
Yeah, I remember your last email, so this agenda already has technical topics as we discussed earlier:

> * Presentation of [realms](https://github.com/tc39/proposal-realms) (15-30 minutes, me)
>   _I assume that after investigation I made for understanding `Membrane` pattern and its usages, I'm qualified enough to talk about `Realms`_
> * Presentation of [pipeline-operator](https://github.com/tc39/proposal-pipeline-operator) (15-30 minutes, me)
> * Presentation of [extended-numeric-literals](https://github.com/tc39/proposal-extended-numeric-literals) (15-30 minutes, me)

Or you were thinking about something else, @littledan?

**Comment by littledan** [comment]
@Igmat Sorry if I'm being confusing; I think it's good that you included technical topics to stay interesting; just keep into account that these sometimes take longer than intended. Maybe the technical topics overall could be timeboxed to 1 hour, just to be safe?

**Comment by Igmat** [comment]
@chicoxyzzy not yet, you may contact me directly (ichulinda@dataart.com) with any questions regarding the event and I'll include our Chief Community Manager in discussion, because he is the one who is responsible for organizational stuff.
We haven't contacted everybody we want to invite yet, but you probably already in the list.
But even if not, I would love you to join us;)

@littledan, ok got it. Could you please clarify, do you mean one hour for each, or one hour for all?

**Comment by js-choi** [comment]
@Igmat: I sent an email reaching out about the pipeline presentation. Please feel free to ask @mAAdhaTTah, @littledan, or I any questions about their details or status when you work on the presentation.

**Comment by littledan** [comment]
@Igmat For all.

**Comment by itchina110** [comment]
I think the official members of TC39 should communicate more than anywhere in the world. Ten years ago, we established a private front-end technology community to promote and promote Web standard technology, and cooperate with W3C China. It has achieved good results. I think TC39 also An office should be set up in China and then coordinated with the civil technology community to collect developer input and feedback.

**Comment by hax** [comment]
@Igmat @littledan 

This is a very great start! I will be very happy see the similar events in China community. I think @itchina110 (who is the organizer of many important tech conferences like FEDay/CSSConf/VueConf/ReactConf etc. in China) is the right person who can help you to organize such events.

**Comment by Igmat** [comment]
I've updated agenda a little bit, with idea that all my talks during this meeting will be interconnected flowing swimmingly from one to another.

Also, if anybody interested, you're welcome to share your ideas about agenda in [google docs](https://docs.google.com/document/d/1wBbuOUuIGf_z6ok-g6AumM2oPrhhAVSs7txmOSrfQcQ/edit?usp=sharing) _I hope you want abuse editing permission_

**Comment by littledan** [comment]
@hax @itchina110 TC39 doesn't have any offices anywhere, but a lot of us in committee would like to improve outreach. I would be very happy to work with you two to set up a similar meeting in China and call into it, just like this one. Should we open a separate issue for planning?

**Comment by Igmat** [comment]
Unfortunately, I have to inform you that this meeting is postponed to summer at least.
There are several reasons behind this situation:
1. We had some organizational issues with time and place - it's major reason.
2. I'll be too busy at April and May due to job and some personal situations that I have to handle.
3. I talked about this initiative with several developers and even though majority of questions is clear for them (understanding of technical part and ways how community can help the committee) still there is one concern: will committee take their efforts into account during decision making process? - I was unable to convince them, since I don't have any examples of affecting committee's decisions by community.

I'm suggesting to use this pause as an opportunity to find a way to guarantee that community's efforts won't be just a huge waste of time - I believe that discussion in #5 is a good starting point.
Even my personal experience show that committee's interaction with community is more like confrontation rather then productive collaboration when **both** sides try to reach common ground.
